Source/monster.cpp

```diff
@@ -50,6 +50,19 @@ void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistribution
 	monst->_mdir = md;
 }
 
+void StartMonsterGotHit(int mid)

```

Comment:
If the above is not possible, I'd suggest renaming to `monsterId` or `monsterIndex`.

Reply:
Renamed to `monsterId` as it was used more often in the existing code base 🙂